ui/lib/core/addon/components/confirm-modal.hbs

```diff
@@ -25,7 +25,7 @@
       {{@disabledMessage}}
     </M.Body>
     <M.Footer as |F|>
```

Comment:
There are now 2 `data-test-cancel` selectors on one file, however they don't show at the same time because of the logic that wraps them. They also do the same thing, though one uses the word "Cancel" and another "Close"....